src/care/array_utils.h

```diff
@@ -181,6 +181,13 @@ CARE_HOST_DEVICE bool checkSorted(const T* array, const int len,
                                   const bool allowDuplicates = false);
 
 
+#if !defined(CARE_ENABLE_IMPLICIT_CONVERSIONS)
```

Comment:
So if we update CHAI, this macro guard will be unnecessary, right?

Reply:
Yes, I added a comment to revisit this when CHAI is updated. It should not be needed if everything is consistent

Reply:
I'm thinking we should just update chai so we don't have to worry about this.

Reply:
Ok. I just double checked and got rid of the guard. care compiles with both implicit conversions on and off. I don't think it will compile with the ale3d develop version of chai (which is what I was using to build ale3d), so both of those will have to updated at the same time for things to work with ale3d.

Reply:
Wait just a sec, don't approve yet

Reply:
Ok, ready for another look after tests pass. Should be fine to get rid of guard now

Reply:
Looks good. Yeah, we should definitely update CHAI at the same time as CARE in the main src.